src/backend.github.js

```diff
@@ -217,7 +217,12 @@ let _ = Mavo.Backend.register($.Class({
 
 							this.repoInfo = repoInfo;
 
-							if (!this.mavo.source) { // if url doesn't have source, check for forks
+							if (this.mavo.storage) { // Check for forks
```

Comment:
Previously, because of this check, the `Switch to my fork` dialog appeared only when an app hadn't got `source`. And I find it not correct. So I made this change.